tests/conftest.py

```diff
@@ -261,6 +265,34 @@ def camera_coefficients(request):
     gc.collect()
 
 
+@pytest.fixture(params=CHECKERBOARD_IMAGES)
+def checkerboard_images(request):
+    yield str(TEST_IMAGES_DIR / request.param)
+    K.clear_session()
+    gc.collect()
+
+
+@pytest.fixture(params=NO_CHECKERBOARD_IMAGES)
+def no_checkerboard_images(request):
+    yield str(TEST_IMAGES_DIR / request.param)
+    K.clear_session()
+    gc.collect()
+
+
+@pytest.fixture(params=DETECTED_CORNERS)
+def detected_corners(request):
+    yield str(TEST_DATA_DIR / "camera_calibration" / request.param)
+    K.clear_session()
+    gc.collect()
+
+
+@pytest.fixture(params=CALIBRATION_DATA)
+def calibration_data(request):
```

Comment:
We should endeavour to keep only the more commonly used fixtures in `conftest.py`, suggest moving all undistortion and camera calibration related fixtures to their respective unit test files instead.

Reply:
fixed